modules/firewall-rules/variables.tf

```diff
@@ -24,27 +24,4 @@ variable "network_name" {
 
 variable "rules" {
   description = "List of custom rule definitions (refer to variables file for syntax)."
-  default     = []
-  type = list(object({
```

Comment:
Why are we removing type info? IIUC to support disabled we can just append to existing rule object?